heartbeat/monitors/active/http/http_test.go

```diff
@@ -808,6 +810,57 @@ func mustParseURL(t *testing.T, url string) *url.URL {
 	return parsed
 }
 
+// helper that compresses some content as gzip
+func gzipBuffer(t *testing.T, toZip string) *bytes.Buffer {
+	var gzipBuffer bytes.Buffer
+	gzipWriter := gzip.NewWriter(&gzipBuffer)
+	defer gzipWriter.Close()
+	_, err := gzipWriter.Write([]byte(toZip))
+	if err != nil {
+		t.Fatal(err)
+	}
+	return &gzipBuffer
+}
+
+func TestDecodesGzip(t *testing.T) {
```

Comment:
Explanatory comment on why we have this test vs. the other would be good

Reply:
https://github.com/elastic/beats/pull/28739/commits/af7937235ddb652ec3a2551b958a0aca1525cb2a